packages/otelbin/src/components/monaco-editor/ResizeBar.tsx

```diff
@@ -22,7 +22,6 @@ export function ResizeBar({ onWidthChange }: ResizeBarProps) {
 		};
 
 		const onMouseMove = (e: MouseEvent) => {
-			e.preventDefault();
```

Comment:
**suggestion:** I believe we should still call `e.preventDefault()` when `state.current.dragging`. This ensures that text selection doesn't happen when resizing the editor.

Reply:
yep, seems to only occur in the error panel. Which is okay.

Reply:
@bripkens hmm it's strange, I tested in Chrome, and there was no extra selecting text in anywhere.